menu_konsolowe.py

```diff
@@ -1,3 +1,66 @@
+from CoWork_klasy import *
+import json
+
+
+def show_admin_desks_view():
+    print("Admin Desks View:")
+    for keys, values in desks_instances.items():
+        print(keys, values)
+
+
+def add_desk():
+    name = input("Podaj nazwę biurka: ")
+    desk_type = input("Podaj typ biurka: ")
+    price = float(input("Podaj cenę biurka PLN/h: "))
+    status = input("Podaj status biurka: ")
+
+    new_desk = Desks(name, desk_type, price, status)
+    desks_instances[name] = new_desk
+    print("Dodano biurko!")
+
+
+def delete_desk():
+    print("Aktualne Biurka:")
+    for idx, (name, desk) in enumerate(desks_instances.items()):
+        print(f"{idx}. {name} - {desk.desk_type} - {desk.price} PLN/h - {desk.status}")
+
+    if not desks_instances:
+        print("Brak dostępnych biurek do usunięcia.")
+        return
+
+    try:
+        del_desk = int(
+            input("Podaj indeks biurka do usunięcia (lub -1, aby anulować): ")
+        )
+
+        if del_desk == -1:
+            print("Anulowano usunięcie.")
+            return
+
+        if 0 <= del_desk < len(desks_instances):
+            desk_name = list(desks_instances.keys())[del_desk]
+            desks_instances.pop(desk_name)
+            print(f"Usunięto biurko: {desk_name}")
+        else:
+            print("Nieprawidłowy indeks. Nie usunięto żadnego biurka.")
+
+    except ValueError:
+        print("Nieprawidłowe dane wejściowe. Podaj prawidłowy indeks.")
+
+
+def save_tasks_to_file():
+    with open("desks.json", "w") as file:
+        data = {}
+        for name, desk in desks_instances.items():
+            data[name] = {
+                "desk_type": desk.desk_type,
```

Comment:
dodałbym do klasy Desk funkcje to_dict, a bardzo mozliwe ze nawet nie trzeba bo `desk.__dict__` chyba zwroci to co potrzebujesz  https://www.javatpoint.com/__dict__-in-python